### PR TITLE
Do not recommend React Celo to Developers

### PR DESCRIPTION
**Pull request by aaronmgdr**
We would like to stop steering devs towards to react-celo package. As such this PR removes all reference to the package from the celo docs. Where appropriate alternatives like viem and rainbowkit are mentioned

Also be sure to mention that one does not need a celo specific sdk to build on celo.

### CODE REVIEW

File: docs/learn/developer-tools.md [hunk 1]
```diff
@@ -23,17 +23,12 @@ Consider using [Dependabot](https://docs.github.com/en/code-security/supply-chai
...
-### react-celo
-
-[react-celo](/developer/react-celo) is [Web3Modal](https://web3modal.com/)-like experience that injects ContractKit into your web-based application. Supports a variety of different wallets, including but not limited to Valora, Ledger, Metamask (Celo compatible fork) and any WalletConnect compatible wallets
-
-<PageRef url="/developer/react-celo" pageName="react-celo" />
 
 ### Ethers.JS Wrapper
 
-A minimal wrapper to make Ethers.JS compatible with the Celo network.
+A minimal wrapper for ethers to create Create celo Easy Fee Transactions. If paying for gas with tokens is not important for your usecase, then the standard ethers.js works perfectly well.
```
Comment (therealharpaljadeja) [inline docs/learn/developer-tools.md:31]: "create Create" -> "create"

File: docs/developer/sdks/celo-sdks.md [hunk 2]
```diff
@@ -9,13 +9,12 @@ import TabItem from '@theme/TabItem';
...
-List of Celo libraries & SDKs
+Because Celo is compitable with Ethereum any ethereum package can be used with Celo. Packages with specific support for Celo include:
 
 ---
 
-- [Celo SDK](https://docs.celo.org/learn/developer-tools#celo-sdk-reference-docs)
-- [thirdweb SDK](../thirdweb-sdk/index.md)
-- [React-Celo](../react-celo/index.md)
+- [viem](../viem/index.md)
 - [ContractKit](../contractkit/index.md)
```
Comment (therealharpaljadeja) [inline docs/developer/sdks/celo-sdks.md:17]: Should we remove `ContractKit` too?

Comment (aaronmgdr) [reply docs/developer/sdks/celo-sdks.md:17]: its ok to keep it

File: docs/learn/developer-tools.md (hunk 1 above)
Comment (aaronmgdr) [inline docs/learn/developer-tools.md:31]:
```suggestion
A minimal wrapper for ethers to create Celo Easy Fee transactions. If paying for gas with tokens is not important for your use case, then the standard ethers.js works perfectly well.
```

**Review by therealharpaljadeja — APPROVED** [review]

**Pull request merged**